esp8266-fastled-webserver/include/configs/controller/controller_template.h

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
SIMPLE FILE REVIEW

The goal of the files in this subdirectory are to provide default
values for each supported MCU.  The MCU-specific files (currently ESP32 and ESP8266) define *default* values for `#define`'d values, including:
* `DATA_PIN`, `DATA_PIN_2`, `DATA_PIN_3`, `DATA_PIN_4`, etc.
* `IR_RECV_PIN`
* `FASTLED_INTERRUPT_RETRY_COUNT`

Critically, these headers will ***NOT*** re-define symbols that are
already defined.  This ensures a product can override the values
as needed.